go.mod

```diff
@@ -4,6 +4,8 @@ go 1.23.0
 
 toolchain go1.23.5
 
+replace github.com/longhorn/types => ../types
```

Comment:
Missed this. Do we need the replacement? @PhanLe1010

Reply:
As mentioned in the PR description, we need to merge this dependent PR first https://github.com/longhorn/types/pull/45 then remove the `replace` directive